src/mtool/ui/src/sagas/storageSaga.js

```diff
@@ -213,7 +213,7 @@ function* fetchDevices(action) {
           alertTitle: "Fetch Devices",
           errorMsg: "Unable to get devices!",
           errorCode: `Description: ${response.data.result && response.data.result.status
-            ? `${response.data.result.status.description}, Error code:${response.data.result.status.code}`
```

Comment:
Just curious. Can I ask why the error code has been deleted?

Reply:
This also comes from internal review. As users are not aware of those error codes,  It might not be useful for them.